src/utils/transaction/userOperation.ts

```diff
@@ -43,12 +45,31 @@ export const generateSignedUserOperation = async (
     address: accountAddress as Address,
   });
 
+  let accountFactoryAddress: Address | undefined;
+
+  try {
+    const cachedFactoryAddress = await redis.get(
+      `account-factory:${accountAddress.toLowerCase()}`,
+    );
+    accountFactoryAddress = getAddress(cachedFactoryAddress ?? "");
```

Comment:
This is indirect to what you want:
- if cachedFactoryAddress exists, set accountFactoryAddress to the checksum version of it
- else accountFactoryAddress is undefined or null

Write the code that way. The current way has some unexpected behavior (what does `getAddress("")` do? You have to read the code to find out):

```typescript
  let accountFactoryAddress: Address | undefined;

  try {
    const cached = await redis.get(
      `account-factory:${accountAddress.toLowerCase()}`,
    );
    if (cached) {
      accountFactoryAddress = getAddress(cached);
    }
  } catch {}
```